clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2000,7 +2000,7 @@ def SYCLIntelMaxGlobalWorkDimAttrDocs : Documentation {
 Applies to a device function/lambda function or function call operator (of a
 function object). Indicates the largest valid global work dimension that will be
 accepted when running the kernel on a device. Valid values are integers in a
-range of [0, 3]. A kernel with max_global_work_dim(0) must be invoked with a
+range of [0, 3]. A kernel with max_global_work_dim(1) must be invoked with a
```

Comment:
`range of [0, 3]` -> `range of [1, 3]` ?

Reply:
No

Reply:
Sorry for the confusion, cause by a missed patch during PR updating.